websites/S/Simulator Radio/dist/metadata.json

```diff
@@ -14,9 +15,9 @@
     "en": "Simulator Radio | Your #1 Simulation Station"
   },
   "url": "simulatorradio.com",
-  "version": "1.4.4",
-  "logo": "https://i.imgur.com/5wpjeAM.png",
-  "thumbnail": "https://i.imgur.com/7OIweId.png",
+  "version": "1.5.0",
+  "logo": "https://i.imgur.com/xjgquwP.png",
+  "thumbnail": "https://i.imgur.com/QIy2Kfe.png",
```

Comment:
Nice logo 
![image](https://user-images.githubusercontent.com/34792838/112739326-d1df1080-8f73-11eb-8bd0-2d10cb233184.png)

Reply:
Hi Bas, sorry about that, not sure why that link went invalid but I've set a new one now